morango/models/utils.py

```diff
@@ -205,6 +205,11 @@ def _mac_is_local(mac):
 
 def get_0_5_mac_address():
 
+    # check whether envvar was set, and use that if available
+    node_id = os.environ.get("MORANGO_NODE_ID")
+    if node_id and len(node_id.strip()) >= 3:
+        return _do_salted_hash(node_id)
```

Comment:
cool, glad we're salting and hashing it

Reply:
> `len(node_id.strip()) >= 3`

maybe we should provide a warning the case that `MORANGO_NODE_ID` exists but is being ignored

Reply:
Opened an issue: https://github.com/learningequality/morango/issues/91